packages/notices/src/store/controls.ts

```diff
@@ -4,7 +4,7 @@
 import { speak } from '@wordpress/a11y';
 
 export default {
-	SPEAK( action ) {
+	SPEAK( action: { message: string; ariaLive?: 'polite' | 'assertive' } ) {
```

Comment:
Is this because `speak()` isn't TS-typed, or because we intentionally want our own source of truth for `action`?

(No opinion, just trying to learn the WPCS standards better)

Reply:
Yeah, that action uses a custom signature.